templates/about.html

```diff
@@ -1,12 +1,12 @@
 {% extends "base.html" %}
+
+{% block header %}<link rel="stylesheet" href="{{ url_for('static', filename='css/abbout.css') }}">{% endblock %}
```

Comment:
spelling mistake on abbout.css
```suggestion
{% block header %}<link rel="stylesheet" href="{{ url_for('static', filename='css/abbout.css') }}">{% endblock %}
```